### ISSUE TITLE
Karaf OSGi project for V19

### ISSUE THREAD

**Issue by QNENet** [vaadin/osgi#19]
More appropriate repo for issue discussed flow issue 9732

@pleku  Although no where near an expert on the inner workings of OSGi I have a reasonable understanding of what a Karaf project should look like. 
I will report back on my findings re cmpn, enterprise and the problem with my initial feature attempt after getting some answers backfrom the experts.

**Comment by QNENet** [comment]
Please note that BJ Hargrave on the OSGi Dev mail list stated

`Also, starting with R8, we will no longer publish osgi.cmpn and osgi.enterprise jars to maven central. So people will need to use the individual specification API jars for R8 specifications.
`

**Comment by denis-anisimov** [comment]
I consider this ticker only as an issue related to 

```
Unable to resolve com.vaadin.flow.osgi/6.0.0.202101100021: missing requirement [com.vaadin.flow.osgi/6.0.0.202101100021]
    osgi.wiring.package;
filter:="(&(osgi.wiring.package=org.osgi.service.http.context)(version>=1.1.0)(!(version>=2.0.0)))"]]
```
And nothing else.

This is a consequence of the fact that Karaf is not compliant to OSGi Compendium 7 specification (and by the way : mentioning R8 in this regard is extremely weird since even R7 is not supported).

But we don't need `1.1.0` version for `org.osgi.service.http.context` package (which is a part of CMPN 7) in fact.
`1.0.0`  is enough and it's supported by Karaf (even version `4.2.x`.

This ticket will be fixed once the PR #24.
Please create another one if you still have other issue with Karaf.
